lib/faker/pizza.ex

```diff
@@ -0,0 +1,162 @@
+
+  @doc """
+  Returns a cheese string
+  """
+  @spec cheese() :: String.t
+  sampler :cheese, ["Mozzarella", "Cheddar", "Feta", "Secret Cheese Blend", "4 Cheese Blend", "Buffalo Mozzarella", "Asiago", "Romano", "Provolone", "Emmental", "Ricotta", "Marscapone", "Swiss", "Parmesan", "Smoked Mozzarella", "Blue (Bleu) Cheese", "Soy Cheese", "Lactose Free Cheese", "Whole Milk Mozzarella", "Burrata", "Goat Cheese", "Cashew Cheese", "Fior di latte", "Paneer", "Cheese Curds", "Gouda", "Fontina", "Gorgonzola", "Queso Fresco", "Crème fraîch"]
+
+  @doc """
+  Returns a meat string
+  """
+  @spec meat() :: String.t
+  sampler :meat, ["Anchovies", "Bacon", "Pulled Pork", "Buffalo Chicken", "Canadian Bacon", "Capicola", "Chicken", "Chorizo", "Clam", "Duck", "Eel", "Egg", "Gyro", "Ham", "Hamburger", "Hot Dogs", "Mackerel", "Meatballs", "Mutton", "Pepperoni", "Philly Steak", "Prosciutto", "Reindeer", "Salami", "Salmon", "Sausage", "Scallops", "Shellfish", "Shrimps", "Smoked Salmon", "Squid", "Steak", "Venison"]
+
+  @doc """
+  Returns a vegetable string
+  """
+  @spec vegetable() :: String.t
+  sampler :vegetable, ["Artichoke Hearts", "Arugula", "Banana", "Banana Peppers", "Basil", "Black Olives", "Broccoli", "Capers", "Caramelised Onions", "Cherry Tomatoes", "Coconut", "Eggplant", "Garlic", "Green Olives", "Green Peas", "Green Peppers", "Habanero Peppers", "Jalapeños", "Mango", "Mushrooms", "Onions", "Oysters", "Peperoncini", "Pickled Ginger", "Pineapple", "Potatoes", "Red Onion", "Red Peppers", "Sauerkraut", "Spinach", "Sun-Dried Tomatoes", "Sweet Corn", "Sweet Potato", "Zucchini"]
+
+  @doc """
+  Returns a size string
+  """
+  @spec combo() :: Strint.t
+  sampler :combo, ["Africana", "All Dressed", "Bacon Cheeseburger ", "BBQ Chicken", "Bianca ", "Bolognese", "Breakfast", "Buffalo Chicken", "Canadian", "Caprese", "Capricciosa", "Capricciosa ", "Cheese", "Chicken Pesto", "Ciao-ciao", "Curry Banana", "Double Dutch", "Fajita", "Fig and Goat Cheese", "Four Seasons", "Frutti di mare", "Funghi", "Grandma", "Greek", "Grilled Vegetarian", "Hawaiian", "Hot & Spicy", "Italian Deli", "Kebab", "Loaded", "Maltija", "Margherita", "Meat Feast", "Meat Lovers", "Meatball ", "Mockba", "Onion & Gorgonzola", "Pepperoni & Mushroom", "Perogie", "Pesto Chicken", "Poutine", "Prociutto Arugala", "Pugliese", "Quattro Formaggi", "Quattro Formaggio", "Quattro stagioni", "Romana", "Shrimp Club", "Smoked Salmon & Goat Cheese", "Supreme", "Taco", "Thai Chicken", "Vegetarian Lovers", "Vegetariana", "Veggie Korma", "Viennese"]
+
+  @doc """
+  Returns a random size or inches
+  """
+  @spec size_or_inches() :: [char]
+  def size_or_inches, do: size_or_inches(:crypto.rand_uniform(1, 3))
+  defp size_or_inches(1), do: size()
+  defp size_or_inches(2), do: inches()
+
+  @doc """
+  Returns a size string
+  """
+  @spec size() :: Strint.t
```

Comment:
I got what you were referring to for most of them, but not sure what the typo is here?